components/deployhq/package.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "@pipedream/deployhq",
+  "version": "0.0.1",
+  "description": "Pipedream DeployHQ Components",
+  "keywords": [
+    "pipedream",
+    "deployhq"
+  ],
+  "license": "MIT"
```

I know the app hasn't been created yet but can you add the following lines?

```suggestion
  "homepage": "https://pipedream.com/apps/deployhq",
  "license": "MIT",
  "publishConfig": {
    "access": "public"
  }
```